remoteappmanager/db/orm.py

```diff
@@ -11,28 +11,35 @@
 Base = declarative_base()
 
 
+class IdMixin(object):
+    """Base class to provide an id"""
+    id = Column(Integer, primary_key=True)
+
+    @classmethod
+    def from_id(cls, session, id):
+        return session.query(cls).filter(cls.id == id).one()
+
+
 class UserTeam(Base):
     """ The user (n <-> n) team association table """
     __tablename__ = "user_team"
     user_id = Column(Integer, ForeignKey('user.id'), primary_key=True)
     team_id = Column(Integer, ForeignKey('team.id'), primary_key=True)
 
 
-class User(Base):
+class User(IdMixin, Base):
```

Comment:
Because then sqlalchemy pretends to have a table for the parent class, and working around it is less smooth than using a mixin like that (which is according to sqlalchemy docs)